Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -145,6 +159,22 @@ - (void)handleRemoteNotificationsRegistered:(NSNotification *)notification
   [self sendEventWithName:@"remoteNotificationsRegistered" body:notification.userInfo];
 }
 
+- (void)handleRegisterUserNotificationSettings:(NSNotification *)notification
+{
+  if (self.requestPermissionsResolveBlock == nil) {
+    return;
+  }
+  UIUserNotificationSettings *notificationSettings = notification.userInfo;
```

Comment:
Oh nice, so it works because I was setting `userInfo` to `UIUserNotificationSettings` instead of setting it a dictionary which contains `UIUserNotificationSettings` like how it's supposed to be used. Bummer. Updated, take a look! Thanks.